cmd/inspector/main.go

```diff
@@ -244,6 +253,47 @@ func NodeInfo(cmd *cobra.Command, args []string) (err error) {
 	return nil
 }
 
+func dotOutput() (*os.File, error) {
```

Comment:
When creating a func that creates something use `New`, `Create` etc. prefix to make clear that it needs to be closed.